components/feature/syncedtabs/src/main/java/mozilla/components/feature/syncedtabs/controller/DefaultController.kt

```diff
@@ -45,4 +49,16 @@ internal class DefaultController(
             }
         }
     }
+
+    /**
+     * See [SyncedTabsController.syncAccount]
+     */
+    override fun syncAccount() {
+        scope.launch {
+            accountManager.withConstellation {
```

Comment:
Not for this PR, but just thinking out-loud:

This makes me think if it's worth consolidating these two things, perhaps as a flag once can pass into `syncNowAsync`.

e.g. this will be a nice entry point:
`syncNowAsync(reason = SyncReason.User, engines = [Tabs], updateConstellation = true)`